.travis.yml

```diff
@@ -4,6 +4,7 @@ rvm:
   - 2.4.1
```

Comment:
Let's remove. We only need to test against one of these minor versions 2.4.5 will work fine.